pkg/ddc/thin/transform_fuse.go

```diff
@@ -151,6 +154,39 @@ func (t *ThinEngine) parseFuseImage(runtime *datav1alpha1.ThinRuntime, value *Th
 	}
 }
 
+func (t *ThinEngine) parseLifecycle(runtime *datav1alpha1.ThinRuntime, profile *datav1alpha1.ThinRuntimeProfile, value *ThinValue) {
+	// default lifecycle config
+	value.Fuse.Lifecycle = &corev1.Lifecycle{
+		PreStop: &corev1.LifecycleHandler{
+			Exec: &corev1.ExecAction{
+				Command: []string{
+					"sh", "-c", fmt.Sprintf("umount %s", t.getTargetPath()),
```

Comment:
To avoid escape risks, it is recommended not to use:
command: ["sh", "-c", "umount {{ .Values.fuse.targetPath }}"]